src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegration.tsx

```diff
@@ -123,17 +122,16 @@ export default class InstalledIntegration extends React.Component<Props> {
     return (
       <Access access={['org:integrations']}>
         {({hasAccess}) => (
-          <Flex align="center" key={integration.id} className={className}>
-            <Box flex={1}>
+          <IntegrationFlex key={integration.id} className={className}>
+            <IntegrationItemBox>
```

Comment:
Not sure if intentional, but `flex={1}` translates to `flex: 1`, but I guess having it as a flexbox el itself accomplishes the same thing?

Reply:
@billyvg So I should have:
```
const IntegrationItemBox = styled('div')`
  flex: 1;
`;
```

Reply:
Yeah that would be the 1:1 conversion

Reply:
@billyvg updated